shared/wallets/transaction-details/index.js

```diff
@@ -51,6 +51,8 @@ export type NotLoadingProps = {|
   transactionID?: string,
   you: string,
   yourRole: Types.Role,
+  // if 'you' is a wallet
+  youWereWallet: string,
```

Comment:
confusing comment+naming here, took me a bit to see 'you' didn't mean `props.you`. Maybe `yourAccountName`?